gomock/controller.go

```diff
@@ -296,3 +296,9 @@ func unwrapTestReporter(t TestReporter) TestReporter {
 	}
 	return tr
 }
+
+// AllExpectedCallsSatisfied returns true in case all expected calls bound to this Controller are satisfied.
```

Comment:
let's move this to be next to other exported functions, closer to top of the file.

Reply:
👍